tests/testdata/fmt/badly_formatted.sql

```diff
@@ -0,0 +1,18 @@
+select * from foo;
```

Comment:
Could you actually move these files to `tests/specs/fmt/sql` instead? I didn't even know we had a test here and I think we should remove these altogether in favor of using spec tests.

Reply:
Is the idea to remove the tests in tests/integration/fmt_tests.rs?

Reply:
Yeah, I would remove them altogether

Reply:
other question, how does one run the spec tests? i tried `cargo test spec::fmt::sql` but did not work i think...

Reply:
Just noticed the README lol

Reply:
`cargo test specs::fmt::sql` will work - notice the "s" in `specs` :)